.github/workflows/tests.yml

```diff
@@ -38,7 +38,11 @@ jobs:
 
       - name: Run tests with coverage
         run: |
-          yarn coverage --ci --reporters github-actions --max-workers ${{ steps.cpu-cores.outputs.count }} ./spec/${{ matrix.specs }}
+          if [[ ${{ github.ref }} == refs/heads/develop ]]; then
```

Comment:
this will probably explode if someone puts quotes in their branch names which seem to be valid, maybe we should use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif